client/cypress/integration/product-list.spec.ts

```diff
@@ -231,7 +230,7 @@ describe ('Add Product to Shopping List', () => {
     page.enterShoppingListCount('1');
     page.clickDialogAddShoppingButton();
     cy.get('.mat-simple-snack-bar-content')
-    .should('contain.text', 'Citrus Organic Hair Rinse, 8 fl oz x1 successfully added to your Shopping List.');
+    .should('contain.text', 'successfully added to your Shopping List.');
```

Comment:
Should this have the `should('be.visible')` added as well?

Reply:
We'll a new PR about tidying up the snackbars and their times.

Reply:
@jpwalbran: Please create that issue as part of closing this or we'll totally forget.

Reply:
See issue #37.